# These are 
#! /bin/sh /usr/share/dpatch/dpatch-run
# Description: Fix some compile warnings that prevent building on Wheezy

@DPATCH@
diff -urNad mysql-cluster-gpl-7.2.4.orig/storage/ndb/test/src/CpcClient.cpp mysql-cluster-gpl-7.2.4/storage/ndb/test/src/CpcClient.cpp
--- mysql-cluster-gpl-7.2.4.orig/storage/ndb/test/src/CpcClient.cpp	2012-02-27 12:00:32.521230852 +0000
+++ mysql-cluster-gpl-7.2.4/storage/ndb/test/src/CpcClient.cpp	2012-02-27 12:06:05.735767614 +0000
@@ -424,7 +424,7 @@
 			  const Properties &args) {
   SocketOutputStream cpc_out(cpc_sock);
 
-  cpc_out.println(cmd);
+  cpc_out.println("%s", cmd);
 
   Properties::Iterator iter(&args);
   const char *name;
